tpm2/tpm2.go

```diff
@@ -250,6 +250,24 @@ func encodePasswordAuthArea(passwords ...string) ([]byte, error) {
 	return concat(size, res)
 }
 
+func encodeAuthArea(sessionHandle tpmutil.Handle, password string) ([]byte, error) {
+	// Empty nonce.
+	var nonce []byte
+	// continueSession set, all other bits clear.
+	attributes := byte(1)
```

Comment:
Can you implement this same as `Flag*` vars?
Something like:

```
type SessionAttributes byte

const (
  AttrContinueSession = SessionAttributes(iota)
  AttrAuditExclusive
  ...
)
```

Reply:
done